test/voteInOrganizationScheme.js

```diff
@@ -0,0 +1,109 @@
+    ],
+    founders: [{
+      address: accounts[0],
+      reputation: web3.toWei(1000),
+      tokens: web3.toWei(1000)
+    }]
+  });
+
+  const schemeToDelete = (await originalDao.getSchemes("ContributionReward"))[0].address;
+  assert.isOk(schemeToDelete);
+  const schemeRegistrar = await originalDao.getScheme("SchemeRegistrar");
+  assert.isOk(schemeRegistrar);
+  const result = await schemeRegistrar.proposeToRemoveScheme(
+    {
+      avatar: originalDao.avatar.address,
+      scheme: schemeToDelete
+    });
+
+  assert.isOk(result.proposalId);
+
+  /**
+       * get the voting machine will be used for this proposal
+       */
+  const votingMachine = await getVotingMachineForScheme(originalDao, schemeRegistrar, 2);
+
+  assert.isOk(votingMachine);
+
+  return { proposalId: result.proposalId, votingMachine: votingMachine.address };
+};
+
+describe("VoteInOrganizationScheme", () => {
+  let dao;
+  let voteInOrganizationScheme;
+  beforeEach(async () => {
+
+    dao = await DAO.new({
+      name: "Skynet",
+      tokenName: "Tokens of skynet",
+      tokenSymbol: "SNT",
+      schemes: [
+        { name: "VoteInOrganizationScheme" }
+      ],
+      founders: [{
+        address: accounts[0],
+        reputation: web3.toWei(1000),
+        tokens: web3.toWei(1000)
+      }]
+    });
+
+    const schemeInDao = await dao.getSchemes("VoteInOrganizationScheme");
+
+    assert.isOk(schemeInDao);
+    assert.equal(schemeInDao.length, 1);
+    assert.equal(schemeInDao[0].name, "VoteInOrganizationScheme");
+
+    voteInOrganizationScheme = await VoteInOrganizationScheme.at(schemeInDao[0].address);
+
+    assert.isOk(voteInOrganizationScheme);
+  });
+
+  it("proposeVote organization vote", async () => {
+
+    const proposalInfo = await createProposal();
+
+    const options = {
+      avatar: dao.avatar.address,
+      originalIntVote: proposalInfo.votingMachine,
+      originalProposalId: proposalInfo.proposalId
+    };
+
+    const result = await voteInOrganizationScheme.proposeVote(options);
+
+    assert.isOk(result);
+    assert.isOk(result.tx);
+    assert.isOk(result.proposalId);
+
+    assert.equal(result.tx.logs.length, 1); // no other event
+    assert.equal(result.tx.logs[0].event, "NewVoteProposal");
+
+    const votingMachine = await getVotingMachineForScheme(dao, voteInOrganizationScheme, 0);
+
+    assert.isOk(votingMachine);
+
+    /**
+         * cast a vote using voteInOrganizationScheme's voting machine.
+         * assert that the proposal is executed.
+         */
+    const tx = await votingMachine.vote(result.proposalId, 1, { from: web3.eth.accounts[0] });
+    // TODO: Update these to use ProposalExecuted
+    // const eventAvatar = Utils.getValueFromLogs(tx, "_avatar", "LogExecuteProposal", 1);
+    // assert.equal(eventAvatar, dao.avatar.address);
+    const eventProposal = Utils.getValueFromLogs(tx, "_proposalId", "LogExecuteProposal", 1);
+    assert.equal(eventProposal, result.proposalId);
```

Comment:
Is that assert the voting in the originalIntVote ?

Reply:
No, it is the current dao.

Reply:
So what does it test ?

Reply:
I'm going to try filtering VoteProposal events for the expected event.

Reply:
Great. It will give the test a meaning :)

Reply:
@orenyodfat 

Thank you for spotting that I hadn't run a test on this.

I am finding that the voter on the original dao's proposal is accounts[0], the current user.

Wouldn't it make sense for that vote to have been done on behalf of the current DAO's avatar?

Reply:
why not, a user can be a member in multiple orgs.

Reply:
this tests suppose to auto run by travis.  are they?.